apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/draft-post-modal/index.js

```diff
@@ -0,0 +1,48 @@
+import { Modal, Button } from '@wordpress/components';
+import { doAction } from '@wordpress/hooks';
+import { useI18n } from '@wordpress/react-i18n';
+import React from 'react';
+import draftPostImage from './images/draft-post.svg';
+import './style.scss';
+
+const DraftPostModal = () => {
+	const { __ } = useI18n();
+	const [ isOpen, setIsOpen ] = React.useState( true );
+	const closeModal = () => setIsOpen( false );
+	const closeEditor = () => doAction( 'a8c.wpcom-block-editor.closeEditor' );
+
+	if ( ! isOpen ) {
+		return null;
+	}
+
+	return (
+		<Modal
+			open={ isOpen }
+			className="wpcom-block-editor-draft-post-modal"
+			onRequestClose={ closeModal }
+		>
+			<div className="wpcom-block-editor-draft-post-modal__image-container">
+				<img src={ draftPostImage } alt={ __( 'Write post', 'full-site-editing' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **44** times:
`translate( 'Write a Post' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/onboarding-draft-post-modal)